README.md

```diff
@@ -73,30 +73,47 @@ Unique methods, not part of `@testing-library/dom`
   - `queryAllByPlaceholderText`
   - `getByPlaceholderText`
   - `getAllByPlaceholderText`
+  - `findByPlaceholderText`
+  - `findAllByPlaceholderText`
   - `queryByText`
   - `queryAllByText`
   - `getByText`
   - `getAllByText`
+  - `findByText`
+  - `findAllByText`
   - `queryByLabelText`
   - `queryAllByLabelText`
   - `getByLabelText`
   - `getAllByLabelText`
+  - `findByLabelText`
+  - `findAllByLabelText`
   - `queryByAltText`
   - `queryAllByAltText`
   - `getByAltText`
   - `getAllByAltText`
+  - `findByAltText`
+  - `findAllByAltText`
   - `queryByTestId`
   - `queryAllByTestId`
   - `getByTestId`
   - `getAllByTestId`
+  - `findByTestId`
+  - `findAllByTestId`
   - `queryByTitle`
   - `queryAllByTitle`
   - `getByTitle`
   - `getAllByTitle`
+  - `findByTitle`
+  - `findAllByTitle`
+  - `queryByDisplayValue`,
+  - `queryAllByDisplayValue`,
+  - `getByDisplayValue`,
+  - `getAllByDisplayValue`,
+  - `findByDisplayValue`,
+  - `findAllByDisplayValue`,
 
 ## Known Limitations
 
-- `waitForElement` method is not exposed. Puppeteer has its own set of wait utilities that somewhat conflict with the style used in `@testing-library/dom`. See [#3](https://github.com/testing-library/pptr-testing-library/issues/3).
```

Comment:
I'm not sure we want to *remove* this note, but maybe convert it to cover `waitForDomChange` and `waitForElementToBeRemoved` as well and nudge to "use the `find*` methods instead".

Reply:
Check latest commit, writing documentation is hard 😁

Reply:
We can't use `find*` queries for `waitForElementToBeRemoved`, but I think we should be able to get that working nicely. Didn't get round to it in this PR but I'll raise an issue if anyone else wants to do it :)

Reply:
sounds good thanks! :)